doc/source/ray-core/ray-dashboard.rst

```diff
@@ -82,57 +314,96 @@ The information of up to 1000 dead actors will be stored.
+
+- The tasks, actors, and placement groups broken down by states.
+- The :ref:`logical resource usage <logical-resources>` across nodes.
+- The hardware resource usage across nodes.
+- The autoscaler status.
+
+See :ref:`System Metrics Page <system-metrics>` for available metrics.
 
 .. note:: 
 
-  The event view feature is experimental.
+  The metrics view required the Prometheus and Grafana setup. **See :ref:`Ray Metrics <ray-metrics>` to learn how to set up Prometheus and Grafana**.
 
-The event view lets you see a list of events associated with a specific type (e.g., autoscaler or job) in a chronological order. The equivalent information is also accessible via CLI commands ``ray list cluster-events`` :ref:`(Ray state APIs)<state-api-overview-ref>`.
+The metrics view lets you view visualizations of the time series metrics emitted by Ray.
 
-There are 2 types of events that are available.
+You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.
 
-- **Job**: Events related to :ref:`Ray job submission APIs <jobs-quickstart>`.
-- **Autoscaler**: Events related to :ref:`Ray autoscaler <cluster-autoscaler>`
+There is also a convenient button to open the grafana UI from the dashboard. The Grafana UI provides additional customizability of the charts.
+
+.. _dash-logs-view:
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/event.png
+Logs view
+---------
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/logs.png
     :align: center
 
+The logs view lets you view all the ray logs that are in your cluster. It is organized by node and log file name. Many log links in the other pages will link to this view and filter the list so the relevant logs appear.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/logs-content.png
+    :align: center
+
+The log viewer provides various search functionality to help find the log messages you are looking for.
+
 Advanced Usage
 --------------
 
+Change the dashboard ports
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tabbed:: CLI
+
+  If you need to customize the port on which the dashboard will run, you can pass
+  the ``--dashboard-port`` argument with ``ray start`` in the command line.
+
+.. tabbed:: Python
+
+  If you need to customize the port on which the dashboard will run, you can pass the
+  keyword argument ``dashboard_port`` in your call to ``ray.init()``.
+
+.. tabbed:: VM Cluster Launcher
+
+  To customize the dashboard port while using the "VM cluster launcher", simply include the "ray start --head --dashboard-port" argument 
+  and specify the desired port number in the "head_start_ray_commands" section of the `cluster launcher's YAML file <https://github.com/ray-project/ray/blob/0574620d454952556fa1befc7694353d68c72049/python/ray/autoscaler/aws/example-full.yaml#L172>`_.
+
+.. tabbed:: Kubernetes
```

Comment:
let's use "KubeRay" to avoid possible confusion? Some people may try to deploy ray on k8s without kuberay

Reply:
I believe this option is available for both kuberay and regular k8s deployment. cc @kevin85421 for confirmation

Reply:
I guess the question here is not about whether it works on general k8s deployment. Instead, it's about what we encourage users to use. We want to emphasize 3 officially recommended ways to deploy a cluster:
- a local head-node-only cluster
- vm launcher
- kubeRay